docs/sovity-edc-api-wrapper.yaml

```diff
@@ -507,6 +507,47 @@ components:
         transportMode:
           type: string
           description: Transport Mode
+        sovereign:
+          type: string
+          description: The publisher and sovereign can be distinct
+        geoLocation:
+          type: string
+          description: Geo location
+        nutsLocation:
+          type: array
+          description: NUTS location
+          items:
+            type: string
+            description: NUTS location
```

Comment:
We should try to refine better texts that can truly explain what each of these fields will contain